### ISSUE TITLE
About the difference between the repo and the paper

### ISSUE THREAD

**Issue by saturosfz** [AntixK/mean-spectral-norm#1]
First, thank the author very much for the great work. 

I read the repo and find that the MeanSpectralNorm applies the multiplier weight ( written as: y = (x - mean(x)) * weight + bias), but in the paper the multiplier does not exist (witten as y = (x - mean(x)) + bias). Is this a bettern implementation or I misunderstood the paper?

Looking forward to your reply.

**Comment by AntixK** [comment]
In the original paper, we did not have any scaling factor. I was trying to check if having a scaling factor (weight in the nn.BatchNorm2d class) has any effect on the performance - renormalizing. (Refer appendix E of the Spectral Normalization for GAN paper.) In my experiments, there wasn't a significant change in performance and the model loses its 1-Lipschitzness. I have kept the code to experiment further.

If interested, check out the cleaner and newer implementation of MSN in another [repo of mine](https://github.com/AntixK/NeuralBlocks/blob/master/blocks/meanspectralnorm.py). This implementation is much faster.
